### DISCUSSION TITLE
cgroup v2 without Systemd

### DISCUSSION THREAD

**Discussion by liyimeng** [opencontainers/runc#3408] [Q&A]
I see many docs talk about configuring cgroup v2 using Systemd driver, almost nothing about running without Systemd. 

In my case, I disable cgroupv1 in kernel and that mount /sys/fs/cgroup with init script, successfully make Containerd believe it is running on top of cgroupv2. However, it seem read out nothing from cgroup, e.g. it keep report zero cpu and memory usage. 

What is the proper way to use cgroupv2 without Systemd?

**Comment by liyimeng** [comment]
I am not sure if this is Containerd problem. I open https://github.com/kubernetes/kubernetes/issues/108364 and https://github.com/k3s-io/k3s/issues/5133 

Adding some debug info there.

**Comment by kolyshkin** [comment]
runc supports both systemd and "fs" (read/write cgroupfs) drivers for cgroup v2.
